### PR TITLE
LPS-165631 [CP 2.0 ] Fixed Quicklink redirection

### PR DESCRIPTION
**Pull request by luisacmn**

### CODE REVIEW

**Comment by luisacmn** [comment]
ci:test:sf

**Comment by luisacmn** [comment]
ci:test:osb-site-initializer-customer-portal

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/journal-articles/actions/journal_article_09.xml [hunk 1]
```diff
@@ -8,6 +8,6 @@
 		<dynamic-content language-id="en_US"><![CDATA[Connect your Liferay DXP to Analytics Cloud to gain insights into customer attributes and behavior.]]></dynamic-content>
 	</dynamic-element>
 	<dynamic-element index-type="keyword" instance-id="iatlbelj" name="ActionUrl" type="text">
-		<dynamic-content language-id="en_US"><![CDATA[https://issues.liferay.com/browse/LPS-140180]]></dynamic-content>
+		<dynamic-content language-id="en_US"><![CDATA[https://support-uat.lxc.liferay.com/web/customer-portal/project/#/{{accountKey}}/activation/analytics-cloud ]]></dynamic-content>
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/journal-articles/actions/journal_article_09.xml:11]:
```suggestion
		<dynamic-content language-id="en_US"><![CDATA[{{siteUrl}}/project/#/{{accountKey}}/activation/analytics-cloud ]]></dynamic-content>
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**